config/samples/org_v2_checluster.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+apiVersion: org.eclipse.che/v2
+kind: CheCluster
+metadata:
+  name: eclipse-che
+  namespace: eclipse-che
+spec:
+  workspaces:
+    defaultNamespace:
+      template: '<username>-che'
+    storage:
+      pvcStrategy: 'common'
+      pvc:
+        claimSize: '10Gi'
```

Comment:
For DWO workspaces, I believe this field is currently a no-op -- DWO has its own configuration for PVC size so unless Che is configuring DWO this will generally be ignored.

I've created https://github.com/devfile/devworkspace-operator/issues/843 to maybe enable something like this in the future.